src/common.ts

```
@@ -0,0 +1,261 @@
+export class LocationTracker {
+  #offset: number = 0;
+  #line: number = 1;
+  #column: number = 0;
+
+  /**
+   * Move the offset and the column number forward {@link n} times.
+   * @param n The number of times to move the offset and column number forward.
+   * @returns The new offset.
+   * @throws Error if {@link n} is less than `0`.
+   */
+  public column(n: number = 1): number {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)